app/controllers/peeps.rb

```diff
@@ -0,0 +1,17 @@
+class Chitter < Sinatra::Base
+  get '/peeps' do
+    @peeps   = Peep.all
+    @replies = Reply.all
+    erb :'peeps/peeps'
+  end
+
+  get '/add_peep' do
+    erb :'peeps/add_peep'
+  end
+
+  post '/add_peep' do
+    peep = Peep.create(content: params[:peep], peeper: current_user.username, time: Time.now.asctime)
```

Comment:
Metrics/LineLength: Line is too long. [101/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)